### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by AlphaNtihinduka**
## Project requirements

- You will find all the details of the Bookstore design in [Zeplin](https://zeplin.io/).
- [Login to Zeplin](https://app.zeplin.io/login) (hint: you will find the credentials in the [sneak peek](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md) of the project).
- Open the project Bookstore CMS.
- Use the information detailed in Zeplin to style the website.
- Your final design should match Zeplin's design.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -24,17 +24,40 @@ const Book = (props) => {
...
+            <p className="percentage">64%</p>
+            <p className="complete">Completed</p>
+          </div>
+        </div>
+
+        <div className="vertical-line" />
+        <div className="right-update">
+          <div className="chapter">CURRENT CHAPTER</div>
+          <div className="chapter-number">CHAPTER 17</div>
+          <div className="progress-btn">Update Progess</div>
```
Comment (prolajumokeoni) [inline src/components/Book.js:54]: Zephin Design          |  Your design     
:-------------------------:|:-------------------------:
image  |  Screenshot_258

- Good job so far 💪🏿 👏🏿 👍🏿 

- [ ] Kindly make the Update progress text in your design to be the same color as the text in the zephin design 👍🏿

File: src/CSS/book.css [hunk 2]
```diff
@@ -4,7 +4,7 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
-  border: 0.1rem solid black;
+  border: 0.1rem solid #888;
```
Comment (prolajumokeoni) [inline src/CSS/book.css]: - [ ] Please use the same border color used in the zephin design, you can also add border-radius so your design looks exactly like the zephin design 😉

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @AlphaNtihinduka, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️

- [ ] The add book component should have the same width with the book component, this will make your design look like the zephin design, please see the screenshot attached in the other comment on your pr 🤝 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @AlphaNtihinduka, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**